plottr/apps/inspectr.py

```diff
@@ -185,14 +190,14 @@ class RunInfo(QtWidgets.QTreeWidget):
     a tree view of that dictionary and display that.
     """
 
-    def __init__(self, parent=None):
+    def __init__(self, parent: Optional[QtWidgets.QWidget] = None):
         super().__init__(parent)
 
         self.setHeaderLabels(['Key', 'Value'])
         self.setColumnCount(2)
 
     @Slot(dict)
-    def setInfo(self, infoDict):
+    def setInfo(self, infoDict: dict) -> None:
```

Comment:
as per signature of `dictToTreeWidgetItems` ?
```suggestion
    def setInfo(self, infoDict: Dict[str, Union[dict, str]]) -> None:
```